langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/AzureOpenAiModelName.java

```diff
@@ -1,47 +0,0 @@
-package dev.langchain4j.model.azure;
-
-/**
- * You can get the latest model names from the Azure OpenAI documentation or by executing the Azure CLI command:
- * az cognitiveservices account list-models --resource-group "$RESOURCE_GROUP" --name "$AI_SERVICE" -o table
- */
-public class AzureOpenAiModelName {
```

Comment:
What about marking the class and/or all the constants as `@Deprecated` instead of deleting them right away? (Someone might be using it)

Reply:
Good point: https://github.com/langchain4j/langchain4j/pull/1205/commits/d09273fe97a06b365f1ef0bac85b2b947d0b462f

Reply:
Nice, thanks! Would be nice to add the `@Deprecated` annotation on the class (now it is only in javadoc), so that users of this class will see the warning in the IDE about the deprecation

Reply:
@agoncal please check my comment above

Reply:
Done. My bad, I thought you were talking about the Javadoc.